boards/arm/96b_wistrio/96b_wistrio.dts

```diff
@@ -39,11 +40,13 @@
 };
 
 &usart1 {
+	pinctrl-0 = <&usart1_tx_pa9 &usart1_rx_pa10>;
 	current-speed = <115200>;
 	status = "okay";
 };
 
-&usart3 {
+&usart2 {
+	pinctrl-0 = <&usart2_tx_pa2 &usart2_rx_pa3>;
```

Comment:
Based on pinmux.c file it would be:
```suggestion
	pinctrl-0 = <&usart2_tx_pa2 &usart2_rx_pa15>;
```
But maybe you fixed it on purpose ?

Reply:
Yes, **usart2_tx_pa15** is not a valid option for stm32l151cb-a.